backend/fms_core/serializers.py

```diff
@@ -273,8 +273,6 @@ class Meta:
         depth = 1
 
 class ProjectSerializer(serializers.ModelSerializer):
-    targeted_end_date = serializers.DateTimeField(read_only=True, format="%Y-%m-%d")
```

Comment:
This was the line that was giving problems.

Reply:
Ok, then I would advise to format it properly in the front-end (I think there are examples in the front-end where dates are formatted to be more human-friendly) otherwise the date is displayed with Z 
![image](https://user-images.githubusercontent.com/1617741/128764527-a3c5cfa4-f207-4e9d-8dbe-c6f95c26b248.png)